improver/calibration/quantile_regression_random_forest.py

```diff
@@ -136,6 +138,18 @@ def prep_feature(
             subset_df[variable_name] = subset_df[variable_name].astype(orig_dtype)
 
         subset_df = subset_df.reset_index()
+
+        if feature_name == "std" and subset_df[variable_name].isnull().any():
+            # If all values are identical for a group, std will be NaN.
+            # Replace these NaNs with 0.
+            subset_df[variable_name] = subset_df[variable_name].fillna(value=0)
```

Comment:
Could this just be, or would it become slower somehow?
```suggestion
        if feature_name == "std":
            # If all values are identical for a group, std will be NaN.
            # Replace these NaNs with 0.
            subset_df[variable_name] = subset_df[variable_name].fillna(value=0)
```